src/clients/save_network.rb

```diff
@@ -247,15 +247,28 @@ def copy_from_instsys
       nil
     end
 
+    # Creates target's default DNS configuration
+    #
+    # It uses DNS configuration as defined in AY profile (if any) or
+    # proposes a predefined default values.
+    def configure_dns
+      ret = false
```

Comment:
"`ret`" means "the value to be returned from this method", so the name is wrong here. `configured` instead?